src/analysis/destruct.new.ml

```diff
@@ -74,7 +74,7 @@ module Predef_types = struct
 end
 
 let placeholder =
-  Ast_helper.Exp.ident (mk_id "??")
```

Comment:
This file is a left-over from when we had several backends, it should be deleted (perhaps as a preliminary PR?)